distribution/src/bin/elasticsearch-service.bat

```diff
@@ -177,6 +180,7 @@ for %%a in ("%ES_JAVA_OPTS:;=","%") do (
   )
   if "!other_opt!" == "true" set OTHER_JAVA_OPTS=!OTHER_JAVA_OPTS!;!var!
 )
+if defined loop_ES_JAVA_OPTS goto :loop
```

Comment:
Can you explain (in a code comment) why this goto is necessary? Shouldn't the loop have already gone through all of the parameters?